meshstack-hub/src/app/app.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+
+@Component({
+  selector: 'app-root',
+  imports: [RouterOutlet],
```

Comment:
d: Please keep in mind. This is a standalone component. We can collect some insights during the development ;)